github/src/wizard/GitHubBranchListStep.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, nonNullProp } from "@microsoft/vscode-azext-utils";
+import { QuickPickItem, l10n } from "vscode";
+import type { GitHubContext } from "../GitHubContext";
+import { loadMoreQp } from "../constants";
+import { Branches, GetBranchesParams, getBranches } from "../wrappers/getBranches";
+
+export class GitHubBranchListStep extends AzureWizardPromptStep<GitHubContext> {
+    private picks: IAzureQuickPickItem<string>[];
+
+    public async prompt(context: GitHubContext): Promise<void> {
+        // We always want fresh picks before prompting in case the user has pressed the back button
+        this.picks = [];
+
+        const placeHolder: string = l10n.t('Select a branch');
```

Comment:
I don't love seeing `l10n.t` everywhere. I think I'd prefer if we made an alias function named `localize` that called `l10n.t`. I think `localize` looks better. Not something we have to change here and now though.

Reply:
![image](https://github.com/microsoft/vscode-azuretools/assets/5290572/91383b0f-2694-4a82-9d29-999409b8bd94)